ape/ape-m1.c

```diff
@@ -124,6 +124,7 @@ struct Syslib {
 #define PT_INTERP                   3
 #define EI_CLASS                    4
 #define EI_DATA                     5
+#define EF_APE_MODERN               1
```

Comment:
1. For code style and readability, could we put this in its own grouping?
2. Shouldn't we choose a different magic number, e.g. `0x03133700`? For example, `EF_SPARCV9_PSO` is `1` and `EF_ALPHA_32BIT` are one. I know no one's probably used this field for a long time, but it's probably better to play it safe, since the `e_machine` (and not the C library) is supposedly the natural owner of this field.

Reply:
Done.